### PR TITLE
Introduce DEPLOY.md

### PR DESCRIPTION
**Pull request by fsat**
Introduce `DEPLOY.md` as the companion to the book `Deploying Reactive Microservices: Strategies for Delivering Resilient Systems`.

The page will describe how to deploy Lagom Chirper example on ConductR sandbox, as well as providing tour of the features provided by ConductR.

This PR is done as part of the first draft to match the deadline of the first draft of the book.

### CODE REVIEW

**Comment by ignasi35** [comment]
Hi @fsat, out of curiosity, is this for internal use or is it meant for end users as a guide for deployment and/or playground to understand Lagom/ConductR better?

File: DEPLOY.md [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+
+#### Rolling upgrade of the app
+
+Performing rolling upgrade on [Production Suite](https://www.lightbend.com/platform/production) is relatively straightforward.
+
+New version of a service can be deployed and run alongside an existing version. Should the new version exposes the same endpoint as the existing, while running alongside each other the traffic from the proxy will be delivered to both new and existing version in round-robin fashion.
+
+In this feature tour, we will be performing a rolling upgrade of the `Friend` service.
+
+There is an instance of `Friend` service already running - look for the bundle with `friend-impl` as its `NAME`. Note the `ID` of the `friend-impl` service - it has `3ff82bf` as its value. We call this identifier "Bundle Id".
```
Comment (ignasi35) [inline DEPLOY.md]: I think `3ff82bf` is used throughout the docs and is a leftover from previous runs. In this case in particular it's mentioned in the paragraph but not listed in the following output sample. If that is correct and iintentional I find it a bit confusing.

Comment (fsat) [reply DEPLOY.md]: @ignasi35 - Thanks for picking this up. I had to restart the sandbox and redeploy when writing this section, so I had to update all the bundle ids. I thought I got all of them, but turns out I missed this one.

I'll be sure to re-read the whole doc and made sure the bundle ids are lined up.

**Comment by fsat** [comment]
@ignasi35 - apologies for the really, really late reply. I just thought no one will come and read it this early! :-)

This is meant the companion to the O'Reilly book `Deploying Reactive Microservices: Strategies for Delivering Resilient Systems`. The deal is Ed Callahan will write the book, while I will prepare the instructions to deploy Lagom into ConductR sandbox along with tour of the feature mentioned in the book as provided by ConductR.

**Comment by ignasi35** [comment]
> I just thought no one will come and read it this early! :-)

I'm member, I get e-mail alerts _all the time_. 😛 
thanks for explaining.

**Comment by edwardcallahan** [comment]
Consider adding an export of CONDUCTR_IP and maybe vpn tunneling from local OSX to AWS (ie want to deploy from a local build out of sbt et al).

Good stuff @fsat! Looking really good overall. Your work here is essential to the success of our book project. Thanks!

It's great to see the SBR demo back up and running, cc: @kikiya

**Comment by edwardcallahan** [comment]
@fsat Shall we discuss the maven user instructions next?

**Comment by fsat** [comment]
@edwardcallahan - I've had discussion with @huntc and @jroper today, and the initial work required is to convert Chirper to maven build, and then introduce a maven equivalent of `sbt-conductr` to build the bundle.

So at the moment I'm working on converting chirper to maven build.

**Comment by edwardcallahan** [comment]
Thanks for the info, Felix!

**Review by huntc — APPROVED** [review]

**Pull request merged**